### PR TITLE
#162505502 Test for modal component:

### PR DESCRIPTION
**Pull request by emmabaye**
#### What does this PR do?
Implement test for modal component
#### Description of Task to be completed?
N/A
#### Where should the reviewer start?
[Modal.test.jsx](https://github.com/AndelaOSP/wire/compare/develop...test-modal-component-162505502?expand=1#diff-cde7a39975bbe3e710731fba1f0053ad)
#### How should this be manually tested?
Run test with comment `npm run test:fend` 
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[162505502](https://www.pivotaltracker.com/story/show/162505502)
#### Screenshots (if appropriate)
N/A

### CODE REVIEW

**Review by theartisancodes — APPROVED** [review]

**Review by theartisancodes — APPROVED** [review]

File: src/Components/Modal/Modal.Component.jsx [hunk 1]
```diff
@@ -93,6 +94,7 @@ class Modal extends Component {
               value={this.state.location}
               onChange={this.handleLocationChange}
               style={styles}
+              id='location-select-field'
```
[the remark this answers is not in the document]
Comment (theartisancodes) [reply src/Components/Modal/Modal.Component.jsx]: has this been fixed?

File: src/Components/Modal/Modal.Component.jsx [hunk 2]
```diff
@@ -80,6 +80,7 @@ class Modal extends Component {
               value={this.state.position}
               onChange={this.handlePositionChange}
               style={styles}
+              id='position-select-field'
```
[the remark this answers is not in the document]
Comment (theartisancodes) [reply src/Components/Modal/Modal.Component.jsx]: has this been resolved?

[the remark this answers is not in the document]
Comment (emmabaye) [reply src/Components/Modal/Modal.Component.jsx]: Yes, it has just been resolved.

File: src/Components/Modal/Modal.Component.jsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (emmabaye) [reply src/Components/Modal/Modal.Component.jsx]: Yes, it has just been resolved.

**Comment by theartisancodes** [comment]
Update the branch so i can merge it @emmabaye

**Comment by emmabaye** [comment]
> Update the branch so i can merge it @emmabaye

I have updated the branch @nzaujk

**Pull request merged**